client/Components/amenities.jsx

```diff
@@ -1,52 +1,66 @@
 import React from 'react';
 
+const amenityIcon = {
```

Comment:
Amazing work getting all of these icons to work. In general, your css looks awesome!